### PR TITLE
COMMERCE-11946 Ensure that serviceContext is providing the correct siteGroupId

### PR DESCRIPTION
**Pull request by dsitu**
https://liferay.atlassian.net/browse/COMMERCE-11946

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/health/status/CommerceGuestCheckoutAuthenticationCommerceHealthStatus.java [hunk 1]
```diff
@@ -102,10 +101,11 @@ public void fixIssue(long companyId, long commerceChannelId)
 			privateLayout = false;
 		}
 
-		ServiceContext serviceContext =
-			ServiceContextThreadLocal.getServiceContext();
+		ServiceContext serviceContext = new ServiceContext();
 
+		serviceContext.setScopeGroupId(commerceChannel.getSiteGroupId());
 		serviceContext.setTimeZone(TimeZone.getDefault());
+		serviceContext.setUserId(PrincipalThreadLocal.getUserId());
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/health/status/CommerceGuestCheckoutAuthenticationCommerceHealthStatus.java:108]: Isn't it better to use User currentUser = _userService.getCurrentUser(); ?
In this case it should be possibile to have an empty PrincipalThreadLocal, but you never know

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:close

**Pull request closed without merging**